frappe/tests/test_image.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+from __future__ import unicode_literals
+
+import unittest
+from PIL import Image
+from frappe.utils.image import strip_exif_data
+import io
+
+class TestImage(unittest.TestCase):
+	def test_strip_exif_data(self):
```

Comment:
This test can be a part of https://github.com/frappe/frappe/blob/develop/frappe/core/doctype/file/test_file.py or https://github.com/frappe/frappe/blob/develop/frappe/tests/test_utils.py. We don't need a new file for a single test.

Reply:
I'll move it to tests_utils.py...

Reply:
Moved:
![image](https://user-images.githubusercontent.com/15175501/100614486-3975ae00-333c-11eb-9793-817868bb0128.png)